frontend-react/src/components/FileHandlers/FileHandlerMessaging.test.tsx

```diff
@@ -182,3 +180,205 @@ describe("FileWarningsDisplay", () => {
         expect(firstCells[3]).toHaveTextContent("first_id");
     });
 });
+
+describe("FileQualityFilterDisplay", () => {
+    test("renders expected content", async () => {
+        const destinations: Destination[] = [
```

Comment:
This brings up a good question about test data and verbosity. Some places we generate fake data, others we have hard-coded stuff in variables like this. Maybe CC @doug-s-nava on this question:  Can we cut this data down to 1-2 destinations?

And maybe more of a follow-up for Doug:  test data generation, yay or nay? I know we're erring on the side of more verbose testing for clarity; does data generation hurt that clarity we're going for?

Reply:
Spoke with Doug and will clean up test data